src/styles/blocks/card.css

```diff
@@ -0,0 +1,101 @@
+.card {
```

Comment:
Add `box-sizing: border-box` property for all elements: in this case paddings and border will be included in card width and it should fix tests